meson_options.txt

```diff
@@ -17,7 +17,9 @@ option('orcc-support', type: 'feature', value: 'auto')
 option('snpe-support', type: 'feature', value: 'auto')
 option('protobuf-support', type: 'feature', value: 'auto')
 option('flatbuf-support', type: 'feature', value: 'auto')
+option('cuda-support', type: 'feature', value: 'auto')
```

Comment:
You don't need cuda-support configured by users.

```
if not get_option('cuda-support').disabled()
```
can be rewritten as
```
if not get_option('tensorrt-support').disabled() or not get_option('tensorrt10-support').disabled()
```

Reply:
Done. (nitpick: this will require any subsequent filter that requires cuda to be added to this condition.)